listener/upload.go

```diff
@@ -355,6 +355,7 @@ func processUpload(account string, host *Host) (*models.SystemPlatform, error) {
 		ModulesList:    processModules(&systemProfile),
 		Basearch:       systemProfile.Arch,
 		SecurityOnly:   false,
+		LatestOnly:		true,
```

Comment:
Wrong formating, will fail on "gofmt error". Anyway, you need to update new version of vmaas client firstly. It's failing on `listener/upload.go:358:3: unknown field 'LatestOnly' in struct literal of type vmaas.UpdatesV3Request`

Reply:
fixed